axelrod/strategies/shortmem.py

```diff
@@ -0,0 +1,36 @@
+from axelrod import Actions, Player
+from axelrod.actions import Action
+
+C, D = Actions.C, Actions.D
+
+class ShortMem(Player):
+    """A player who only ever defects."""
```

Comment:
I don't think this docstring is accurate right?

Also note that we require a few more details in the docstring, you can see an example here: http://axelrod.readthedocs.io/en/latest/tutorials/contributing/strategy/writing_the_new_strategy.html